testinfra/modules/service.py

```diff
@@ -173,7 +173,14 @@ def is_running(self):
 
     @property
     def is_enabled(self):
-        raise NotImplementedError
+        # rcctl ls is only available on 5.8 or newer
+        if self.check_output('uname -r') >= '5.8':
+            for service in self.check_output("rcctl ls on").splitlines():
+                if service and service == self.name:
+                    return True
+            return False
+        else:
+            return NotImplementedError
```

Comment:
Thanks for adding this ! I tested and it work well !

I'd remove the check on OpenBSD version, I think it's ok to let the command fail on older versions.
Also the loop seems useless, you can write ``return self.name in self.check_output("rcctl ls on").splitlines()``

An other issue with this implementation is that a service that not exists will always been reported as 'not is_enabled'. For instance this could occur by a typo in the service name, like ssh instead of sshd. I'd expect an error here.

Something like:

```python
@property
def is_enabled(self):
    if self.name in self.check_output('rcctl ls on').splitlines():
        return True
    if self.name in self.check_output('rcctl ls off').splitlines():
        return False
    raise RuntimeError('could not determine if {0} is enabled, does this service exist ?'.format(self.name))
```